devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
+
+NOTE: The columns in the catalog are those expected by galsim.RealGalaxyCatalog.
+However, some parts of that analysis, i.e parametric fits, were not performed.
+Hence those values were manually set and do not have physical meaning.
+
+(1) The catalogs themselves are in AEGIS_galaxy_catalog_[VI]_25.2.fits (where
+[VI] indicates either V or I).  These files can be used to initialize a
+galsim.RealGalaxyCatalog or galsim.COSMOSCatalog instance, and contain a FITS
+tables that could also be  read in directly using, e.g., python or IDL.  The
+fields that are included for each galaxy are:
+
+IDENT: AEGIS object identifier. This value is internally derived.  Therefore
+do not rely on this IDENT for cross-matching between other AEGIS catalogs.
+
+RA, DEC: Right ascension and declination (J2000, degrees).  While this is not
+needed for image simulation, it is necessary for proper cross-matching with
+other catalogs.
+
+MAG: magnitude (MAG_AUTO calculated from sextractor).
+
+BAND: Passband for the observed image (F814W/F606W).
+
+WEIGHT: A weight factor to use to account for the fact that larger galaxies are
+preferentially excluded from this catalog due to the proximity to CCD edges.
+(Currently all weights are set to 1.0).
```

Comment:
The weight was computed as follows:

- We took our original catalogs based on sextractor and applied whatever selection criteria were used to decide whether we wanted to make a postage stamp (e.g., if you only have a magnitude cut, this would just be a magnitude cut).  This gives us what we call the "parent sample".
- We then took the galaxies for which we actually made postage stamps.  Some galaxies didn't get postage stamps because the stamp would have overlapped with the edges of the individual COSMOS pointings, which would have led to funny image artifacts.  This is the "postage stamp sample".  Ideally it's the same as the parent sample, but for the nuisance effects that make us choose not to make a postage stamp.
- Plot the histogram of flux radius for the parent sample and the postage stamp sample.  The ratio of these is the weight, i.e., we had a deficit of high flux radius objects and that's why the weight is increased for those.

The reason we only did this for flux radius is that the postage stamp size had an explicit flux radius-dependence, and so we knew that if any parameter was going to be a problem, it would be that one.